browser/ts/index.ts

```diff
@@ -271,13 +271,13 @@ export interface OrderInfo {
 
 interface WrapperGetOrdersResponse {
     snapshotID: string;
-    snapshotTimestamp: number;
+    snapshotTimestamp: string;
     ordersInfos: WrapperOrderInfo[];
 }
 
 export interface GetOrdersResponse {
     snapshotID: string;
-    snapshotTimestamp: number;
+    snapshotTimestamp: Date;
```

Comment:
This is technically a breaking change. I'm okay with merging for now but let's update the changelog later before we release 9.0.0.